tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventParseMethodContract.scala

```diff
@@ -87,7 +87,12 @@ trait LinagoraCalendarEventParseMethodContract {
            |        "parsed": {
            |            "$blobId": {
            |                "title": "Sprint planning #23",
-           |                "description": "description 123"
+           |                "description": "description 123",
+           |                "start": "2017-01-11T16:00:00",
+           |                "end": "2017-01-11T17:30:00",
```

Comment:
Should be `utcStart`, `utcEnd` cf https://jmap.io/spec-calendars.html#calendar-events

This will make it easier for the front to express the event into the user timezone.

Reply:
hence we don't need `timeZone` property in the response payload?

Reply:
no, put `end` `start` `utcEnd` and `timezone`

Reply:
and `utcStart` :)